### PR TITLE
Adjusting self-report output data formatting

### PR DESCRIPTION
**Pull request by paulosoaresua**
- Added more 2 button choices to the last self-report question
- Formatted the output of the responses to comply with the pattern we designed
- Fixed error that happened when no self-report folder existed

### CODE REVIEW

File: tools/run_session [hunk 1]
```diff
@@ -141,8 +140,9 @@ fi
 # tutorial and main missions.
 sleep 6
 
-# Get rid of any pre-existing discrete events JSON files.
+# Get rid of any pre-existing discrete events and self-reports JSON files.
 $rm -f "${TOMCAT}"/external/malmo/Minecraft/run/saves/discrete_events/discrete_events.json
+rm -f "${TOMCAT}"/external/malmo/Minecraft/run/saves/self_reports/*.json
```
Comment (adarshp) [inline tools/run_session:145]: @paulosoaresua can you change this back to `$rm` or to `/bin/rm`? On some computers (e.g. Emily's) `rm` is aliased to something else.

Comment (paulosoaresua) [reply tools/run_session]: Oh, I din't realize that. Issue fixed.

Comment (paulosoaresua) [reply tools/run_session]: @adarshp, I cannot see the details of the error in the test.

Comment (adarshp) [reply tools/run_session]: Not sure what happened - the MacOS test timed out- I’ll merge it.

Comment (paulosoaresua) [reply tools/run_session]: Ok. I ran it in my machine with no problems.

**Pull request merged**